.github/ISSUE_TEMPLATE/ci-sev.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: "⚠️ CI SEV"
 about: Tracking incidents for PyTorch's CI infra.
+labels: "ci: sev"
```

Comment:
Learn from https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository that we can attach labels automatically when creating the issue.